src/components/Card/CardAvatar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import cardAvatarStyle from '../../assets/styles/components/cardAvatarStyle';
+
+function CardAvatar({ ...props }) {
+  const { classes, children, className, plain, profile, ...rest } = props;
+  const cardAvatarClasses = classNames({
+    [classes.cardAvatar]: true,
+    [classes.cardAvatarProfile]: profile,
+    [classes.cardAvatarPlain]: plain,
+    [className]: className !== undefined
```

Comment:
You could just add another parameter to the `classNames` function and pass in `className`.

Reply:
that function is from a[ NPM package](https://www.npmjs.com/package/classnames)

Reply:
Yeah, I know. You could just call it like this instead `classNames({ ...configObject}, className)`, unless `className` is a bool instead of a string.

Reply:
Ok, I'll change that